src/pathOr.js

```diff
@@ -0,0 +1,30 @@
+var _curry3 = require('./internal/_curry3');
+
+
+/**
+ * If the given, non-null object has an own property at the given path, returns
+ * the value at that path. Otherwise returns the provided default value.
```

Comment:
We're doing `val[paths[idx]]`, so the property needn't be an *own* property. Let's update the description accordingly.